atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -209,12 +209,7 @@ public void testClassicWriteSkew() {
         withdrawMoney(t2, false, false);
 
         t1.commit();
-        try {
-            t2.commit();
-            fail();
-        } catch (TransactionSerializableConflictException e) {
-            // this is expected to throw because it is a write skew
-        }
```

Comment:
This is correct, though I'd add an `as()` clause for the explanation

Reply:
I believe it uses the message (if present) from `fail(String)`. Losing the comment is unfortunate, we had tried to preserve them in some refactors, but clearly not all cases :-/

Reply:
It does, yep! I think you can read most of these as comments I'm writing to myself to fix.

I don't know if you want things of the form `try { <things>; fail(); } catch (Exception e) { // INCLUDEME }` since I'm sure there are going to be quite a few that just say `expected` or things like that

Reply:
Ya that's true, at the end of the day we always want humans to review this sort of change. If you see a pattern that occurs frequently and isn't difficult to detect, use your judgement on filing a ticket to track :-)